tests/qunit/setup-elementor-common.js

```diff
@@ -5,4 +5,5 @@ const elementorCommonConfig = {
 	ajax: { url: '-' },
 	i18n: {},
 	urls: {},
+	rest_url: 'http://localtest/wp-json/',
```

Comment:
Move it as mentioned above

Reply:
Handled.

Reply:
https://github.com/elementor/elementor/pull/11388/commits/bbdfa741e9ca7b8f7231f161928a04c97c012938